src/ui/views/Roster/TopStuff.tsx

```diff
@@ -134,6 +135,24 @@ const TopStuff = ({
 			<div className="d-flex mb-3">
 				<div className="team-picture" style={logoStyle} />
 				<div>
+					<div className="btn-group mb-3">
+						<a href={helpers.leagueUrl(["roster", `${abbrev}`, season - 1])}>
```

Comment:
In Bootstrap, you can put the `btn` classes right on the `<a>` tag https://getbootstrap.com/docs/4.5/components/buttons/#button-tags and then it'll play nicer with `btn-group` too.